dotnet-471-dev-pack/README.md

```diff
@@ -0,0 +1,107 @@
+# Habitat package: dotnet471
+
+## Description
+
+This package is for the .NET 4.7.1 framework developer pack from microsoft. It provides 4.7.1 reference assemblies as well as the SDK needed to support building .NET 4.7.1 applications
+
+## Maintainers
+
+The Habitat Maintainers humans@habitat.sh
+
+## Type of Package
+
+Binary
+
+## Usage
+
+Add core/dotnet471 to pkg_build_deps then leverage nuget and msbuild to build and deploy your application. For an example see mwrock's asp full framework MVC blog post: https://www.habitat.sh/blog/2017/08/Habitat-plans-for-net-full-iis-apps/
+
+# Testing dotnet-471-dev-pack on Windows
+
+**Note**: The following manual steps can be performed in one automated step: [see these instructions](#run-the-pester-tests-via-powershell-script)
+
+## Build and Install
+
+### Build the dotnet-471-dev-pack core plan
+
+Ensure that a terminal is opened to the root directory of the repo
+
+For example
+
+```powershell
+# build dotnet-471-dev-pack from its own directory
+hab pkg build .\dotnet-471-dev-pack
+```
+
+## Install and run the tests for the dotnet-471-dev-pack core plan
+
+### Manually install and run the Pester tests
+
+First ensure that the terminal is located in the core-plans directory and then open a dockerised habitat studio.  
+
+```powershell
+hab studio enter -D
+```
+
+Staying within the same habitat studio environment install the dotnet-471-dev-pack core plan
+
+```powershell
+# Load the habitat build variables
+. .\results\last_build.ps1
+
+# install habitat package
+hab pkg install .\results\$pkg_artifact
+```
+
+Now run the Pester tests.  **Note** if the Pester module is not already installed in the test environment, then run something like ``Install-Module Pester -SkipPublisher -Force`` before proceeding.

```

Comment:
Same comment as below in the test script; these flags might be seen as risky and according to https://github.com/pester/Pester/wiki/Installation-and-Update aren't always needed. I would just hyperlink to that page

Reply:
Adjusted README to only reference the Pester URL instead of recommending an installation statement